examples/raja-teams.cpp

```diff
@@ -227,14 +227,25 @@ int main()
     // Upper triangular pattern
     //========================
     const int N_tri = 5;
+    using team_t = RAJA::TeamExclusive<5>;
```

Comment:
Here is a sketch for doing thread exclusive memory. It does require users knowing upper bounds of the team 
dimensions